examples/models/latex_labels.py

```diff
@@ -1,5 +1,5 @@
-''' A plot showing LaTeX ``Label`` objects in many different locations,
-inside and outside the figure.
+''' A plot showing LaTeX ``Label`` objects in many different point,
```

Comment:
```suggestion
''' A plot showing LaTeX ``Label`` objects in many different points,
```

Maybe "positions" could be another option.